### PR TITLE
fix: Updated hindi locale with missing messages.

### PR DESCRIPTION
**Pull request by jaideepghosh**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Updated hindi locale with missing messages.      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by jaideepghosh** [comment]
@afc163 I'm not sure why the above checks are failing. FYI, I've added hindi translations. Please review the PR and let me know if any change is needed.

**Comment by afc163** [comment]
@jaideepghosh https://www.githubstatus.com/

**Comment by yoyo837** [comment]
Maybe adding the issue template should create another PR.

**Comment by jaideepghosh** [comment]
Thanks @yoyo837 Reverted the issue template change from this PR.
I've created a new PR [30542](https://github.com/ant-design/ant-design/pull/30542) for the Issue Template.

**Comment by yoyo837** [comment]
#23369

**Comment by yoyo837** [comment]
Please update snapshot.

**Comment by jaideepghosh** [comment]
Hey @yoyo837 
What do you mean by `Please update snapshot.`
I'm not sure about this process.

**Comment by yoyo837** [comment]
Pls run code `npm run test -- -u` and push all changes.

File: components/progress/__tests__/__snapshots__/demo.test.js.snap [hunk 1]
```diff
@@ -849,7 +849,7 @@ Array [
           stroke="url(#ant-progress-gradient-13)"
           stroke-linecap="round"
           stroke-width="6"
-          style="stroke:[object Object];stroke-dasharray:295.3097094374406px 295.3097094374406px;stroke-dashoffset:-0px;transition:stroke-dashoffset .3s ease 0s, stroke-dasharray .3s ease 0s, stroke .3s, stroke-width .06s ease .3s, opacity .3s ease 0s"
```
Comment (yoyo837) [inline components/progress/__tests__/__snapshots__/demo.test.js.snap:852]: Some other commits are mixed in, please wait for #30556 to be merged, and then rebase this.

Comment (jaideepghosh) [reply components/progress/__tests__/__snapshots__/demo.test.js.snap:852]: @yoyo837 Please check now.

Comment (yoyo837) [reply components/progress/__tests__/__snapshots__/demo.test.js.snap:852]: You have not do `rebase`.

**Pull request merged**